app/services/qa/linked_data/authority_url_service.rb

```diff
@@ -2,59 +2,43 @@
 module Qa
   module LinkedData
     class AuthorityUrlService
-      # Build a url for an authority/subauthority for the specified action.
-      # @param authority [Symbol] name of a registered authority
-      # @param subauthority [String] name of a subauthority
-      # @param action [Symbol] action with valid values :search or :term
-      # @param action_request [String] the request the user is making of the authority (e.g. query text or term id/uri)
-      # @param substitutions [Hash] variable-value pairs to substitute into the URL template
-      # @return a valid URL the submits the action request to the external authority
-      def self.build_url(action_config:, action:, action_request:, substitutions: {}, subauthority: nil)
-        action_validation(action)
-        url_config = Qa::IriTemplate::UrlConfig.new(action_url(action_config, action))
-        selected_substitutions = url_config.extract_substitutions(substitutions)
-        selected_substitutions[action_request_variable(action_config, action)] = action_request
-        selected_substitutions[action_subauth_variable(action_config, action)] = action_subauth_variable_value(action_config, subauthority, action) if subauthority.present?
-
-        Qa::IriTemplateService.build_url(url_config: url_config, substitutions: selected_substitutions)
-      end
+      class << self
```

Comment:
Used to make class easier to read by removing the need to mark every private method using private_method_class.